app/test/unit/git/squash-test.ts

```diff
@@ -101,47 +101,50 @@ describe('git/cherry-pick', () => {
 
   it('squashes multiple commit non-sequential commits (reorders, non-conflicting)', async () => {
```

Comment:
This refactor did not break this test. I just decided to enhance it to verify that the change made in the refactor does in fact result in the squashing commits being order with respect to the log.